_posts/2019-02-07-partner-api-status-endpoint.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title:  "Partner API Status Endpoint"
+date:   2019-02-07 4:00:00 -0400
+categories: api
+author: Kyle Chan
+---
+
+*Now that you have submitted your first deal via our [Lead Submission Endpoint](http://tech.forwardfinancing.com/api/2017/03/07/using-our-lead-submissions-api.html),
+you're probably wondering how it's doing.*
+
+## Getting Started
+The only things you'll need are: 
+- The lead `id` you received when you first submitted your deal
+- Your `api_key`
+
+### Authentication
+
+You will need to contact us to get your `api_key`. The API key should be included
+in your request headers such that the request header key is `api_key` and the
+request header value for that key is your `api_key`. If you omit the api_key or
+provide an incorrect one, the HTTP status code of the response will be 401
+Unauthorized.
```

Comment:
I think this section is a little confusing with the use of `api_key` to refer to both the specific request header key and the API key itself. Since the example below is so much clearer, we might consider simplifying this section to just say something like:

"The API key should be included in your request headers. If it is missing or incorrect, the HTTP status code of the response will be 401 Unauthorized. See the request example below for the correct formatting of this request header."

Reply:
Ahh yeah, that's confusing